bears/c_languages/CPPCheckBear.py

```diff
@@ -19,6 +19,13 @@ class CPPCheckBear:
     """
 
     LANGUAGES = {"C", "C++"}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'https://groups.google.com/forum/#!forum/coala-devel'}
+    MAINTAINERS = {'Lasse Schuirmann', 'Fabian Neuschmidt', 'Mischa Kr\xfcger'}
```

Comment:
this is what the current setup.py says .. you want me to make it equal to authors if not given? in Bear.py?